sda-commons-server-kafka/build.gradle

```
@@ -19,5 +19,10 @@ dependencies {
   testImplementation 'org.assertj:assertj-core'
 
   testImplementation project(':sda-commons-server-kafka-testing')
+
+  test {
+    // running one test at time, to avoid problems with kafka servers concurrency
+    maxParallelForks = (int) (Runtime.runtime.availableProcessors() / 2 + 1)
```

The code and the comment looks weird together. Wouldn't _one test at time_ be like this?

```suggestion
    maxParallelForks = 1
```